cmd/cli/kubectl-kyverno/commands/apply/command.go

```diff
@@ -163,9 +165,14 @@ func (c *ApplyCommandConfig) applyCommandHelper(out io.Writer) (*processor.Resul
 	if err != nil {
 		return rc, resources1, skipInvalidPolicies, responses1, err
 	}
-	exceptions, err := exception.Load(c.Exception...)
-	if err != nil {
-		return rc, resources1, skipInvalidPolicies, responses1, fmt.Errorf("Error: failed to load exceptions (%s)", err)
+	var exceptions []*kyvernov2beta1.PolicyException
+	if c.inlineExceptions {
+		exceptions = exception.SelectFrom(resources)
```

Comment:
I think it would be better to consider exceptions as a part of the policies path provided in the CLI.

Reply:
This does not work for me, as I have global available company policy file, that is used in all ci pipelines.
Exceptions should not be defined on a global level, they if needed should be in a specific helm chart that is tested.

Reply:
How about we do both? If exceptions are found on the policies path, use them. In addition process the policyexceptions path if provided.

Reply:
Yes, that what I was thinking about. It makes sense to have exceptions on the policies path not the resources path.

Reply:
> Yes, that what I was thinking about. It makes sense to have exceptions on the policies path not the resources path.

@MariamFahmy98  I still disagree here.
The reason for that PR is that we have company wide policies built in in CI pipelines as a helm Plugin. With the goal to to male akk carts compmuant to our standards. Hence the policies are static and fol all the same. 
If one of these charts uses a dependency to a 3rd patty chart, that can not comply to these policies. The chart needes to implement an exception for only this use case in orfer to pass the check. Since it is all part of the same helm chart resources, the exceptions are part of the resources to be checked. Having the exceptions separate makes these checks much more complicated for our developers.

The helm plugin piped the output of `helm template` to `kyverno apply` with the global policy file as argument.

Reply:
Seems like there are three options to specify exceptions via the CLI:
1. Via a separate flag, as implemented today
2. As resources in the policies path
3. As resources in the resources path

Since a design goal for exceptions was to allow developers to request these, it does make sense to allow adding them to the resource paths, much like in a cluster developers can add to namespaces.

For security reasons, when running as an admission controller Kyverno allows configurations of namespaces where exceptions can be defined.

@bakito - is there a concern that dev teams can simply add exceptions to their chart, to bypass policy checks? Is there a way you can enforce approvals or some check on what exceptions are allowed?

Reply:
> Seems like there are three options to specify exceptions via the CLI:
> 
> 1. Via a separate flag, as implemented today
> 2. As resources in the policies path
> 3. As resources in the resources path
> 
> Since a design goal for exceptions was to allow developers to request these, it does make sense to allow adding them to the resource paths, much like in a cluster developers can add to namespaces.
> 
> For security reasons, when running as an admission controller Kyverno allows configurations of namespaces where exceptions can be defined.
> 
> @bakito - is there a concern that dev teams can simply add exceptions to their chart, to bypass policy checks? Is there a way you can enforce approvals or some check on what exceptions are allowed?

@JimBugwadia Yes, apps with exceptions need special permissions to be rolled out. By default exceptions are blocked by argocd in our case.
But I get your point and concerns.

Reply:
Ok, and I assume that in your clusters you are allowing exceptions to be in any namespace.

In this case, although its not much protection, keeping the new flag may be good as it requires an additional consent. 

Minor suggestion on naming and help:

```go
cmd.Flags().BoolVarP(&applyCommandConfig.inlineExceptions, "exceptions-with-resources", "", false, "Evaluate policy exceptions from the path to resource files")
```

Thanks @bakito!

Reply:
@JimBugwadia thank you. I changed the flag name accordingly in the code and docs.

Reply:
Thanks @bakito! Can you resolve conversations if all are addressed?